<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDoctorDialog</class>
 <widget class="QDialog" name="AddDoctorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="doctor_ID_number_label">
       <property name="text">
        <string>ID Number</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="doctor_name_label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="doctor_surname_label">
       <property name="text">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="doctor_speciality_label">
       <property name="text">
        <string>Speciality</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="doctor_email_label">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="doctor_ID_number_line"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="doctor_name_line"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="doctor_surname_line"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="doctor_speciality_line"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="doctor_email_line"/>
     </item>
     <item row="5" column="1">
      <widget class="QDialogButtonBox" name="doctor_button_box">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
